src/shogun2-util/src/main/java/de/terrestris/shogun2/util/interceptor/OgcMessage.java

```diff
@@ -0,0 +1,393 @@
+package de.terrestris.shogun2.util.interceptor;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+/**
+ *
+ * @author Daniel Koch
+ * @author terrestris GmbH & Co. KG
+ *
+ */
+public class OgcMessage {
+
+	/**
+	 * The OGC service type, e.g. WMS, WFS or WCS.
+	 */
+	private String service;
+
+	/**
+	 * The OGC operation type, e.g. GetMap.
+	 */
+	private String operation;
```

Comment:
These Is this (semantically) the same property as in the `InterceptorRule`?

Maybe it would be more "clean" to use an Enum here (for example as we do [here](https://github.com/terrestris/shogun2/blob/master/src/shogun2-core/shogun2-model/src/main/java/de/terrestris/shogun2/model/module/NominatimSearch.java#L46)) :wink: